hmcts/pre-portal
# S28 3553/status page

### JIRA ticket(s)

- S28-3553
- S28-3559
- S28-3560

### Change description

- added stats page to display status of pre components
  - api
  - portal
  - cvp
  - mediakind



> [!IMPORTANT]
> This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

**QA instructions**

- login to the portal as a super user
- go to `/admin/status`
- check the status of componenets
